tasks/managed/push-rpms-to-pulp/push-rpms-to-pulp.yaml

```diff
@@ -226,6 +234,28 @@ spec:
 
         echo "Detected architectures: ${ARCHES[*]}"
 
+        # Check if we have any noarch RPMs that need to be uploaded
+        noarch_rpms=()
+        for file in "${FILES_TO_UPLOAD[@]}"; do
+          if [[ "$file" == *.noarch.rpm ]]; then
+            noarch_rpms+=("$file")
+          fi
+        done
+
+        # If we have noarch RPMs but no architectures, use default architectures from parameter
```

Comment:
should probably add a unit test for this case, right?

Reply:
added a test